edx_when/models.py

```diff
@@ -95,20 +95,18 @@ def schedule_for_user(self, user):
         Return the schedule for the supplied user that applies to this piece of content or None.
         """
         no_schedules_found = None
+        if Schedule is None:
+            return no_schedules_found
         if isinstance(user, int):
-            if Schedule is None:
-                return no_schedules_found
             try:
                 return Schedule.objects.get(enrollment__user__id=user, enrollment__course__id=self.course_id)
-            except Schedule.DoesNotExist:
```

Comment:
nit: I think this can be simplified. 
```
kwargs = {'enrollment__course__id':self.course_id}
if isinstance(user, int):
    kwargs['enrollment__user__id'] = user
else:
    kwargs['enrollment__user__id'] = user.id
try:
    return Schedule.objects.get(**kwargs)
except ObjectDoesNotExist: 
    return no_schedules_found
```

Reply:
Yes, but my expectation is that in the future we might want to use pre-fetched schedules, which is why I kept the parallel structure.